### DISCUSSION TITLE
Erro no teste de consulta status sefaz

### DISCUSSION THREAD

**Discussion by LCASAGRA1** [TadaSoftware/PyNFe#211] [Q&A] [answered]
Boa tarde!

Estou utilizando o exemplo para verificar o status da sefaz desta forma:
from pynfe.processamento.comunicacao import ComunicacaoSefaz

certificado = "/Users/brtec/Documents/GitHub/PyNFe/tests/certificado.pfx"
senha = '123456'
uf = 'pr'
homologacao = True

con = ComunicacaoSefaz(uf, certificado, senha, homologacao)
xml = con.status_servico('nfe')
print(xml.text)

Mas recebo o seguinte erro:
SSLError: HTTPSConnectionPool(host='homologacao.nfe.sefa.pr.gov.br', port=443): Max retries exceeded with url: /nfe/NFeStatusServico4 (Caused by SSLError(SSLError(399, '[SSL: EE_KEY_TOO_SMALL] ee key too small (_ssl.c:3874)')))

Alguém poderia me ajudar?
Luiz

**Comment by leogregianin** [comment, marked as answer]
Bom dia @LCASAGRA1, de acordo com a mensagem de erro, o arquivo do certificado digital que está utilizando não parece ser válido. 

Se está utilizando o arquivo pfx que está em [/tests/](https://github.com/TadaSoftware/PyNFe/tree/main/tests), ele foi criado somente para testes da biblioteca e não é válido para uso nos webservices.

Testei aqui com um certificado digital válido e funcionou normalmente.

**Reply by LCASAGRA1** [reply]
Obrigado pela atenção @leogregianin.
